torch/csrc/Module.cpp

```diff
@@ -461,10 +463,12 @@ PyObject *THPModule_deterministicCuDNN(PyObject *_unused, PyObject *noargs)
 
 PyObject *THPModule_setDeterministicAlgorithms(PyObject *_unused, PyObject *arg)
 {
+  HANDLE_TH_ERRORS
   THPUtils_assert(PyBool_Check(arg), "use_deterministic_algorithms expects a "
           "bool, but got %s", THPUtils_typename(arg));
   at::globalContext().setDeterministicAlgorithms(arg == Py_True);
   Py_RETURN_NONE;
+  END_HANDLE_TH_ERRORS
 }
```

Comment:
These functions were missing the wrappers that turn the warning into a python warning, so the test was failing after replacing `maybeWarnRegex` with `assertWarnsOnceRegex`

Reply:
Nice catch.

Reply:
cc @kurtamohler

Reply:
Thanks for fixing that!